browser-client.cpp

```diff
@@ -19,6 +19,7 @@
 #include "browser-client.hpp"
 #include "obs-browser-source.hpp"
 #include "base64/base64.hpp"
+#include "..\obs-websocket\lib\obs-websocket-api.h"
```

Comment:
This header must not be include this way.

Reply:
Added a copy of the header file now